src/Setoid.ts

```diff
@@ -58,3 +60,13 @@ export const getProductSetoid = <A, B>(SA: Setoid<A>, SB: Setoid<B>): Setoid<[A,
     equals: ([xa, xb], [ya, yb]) => SA.equals(xa, ya) && SB.equals(xb, yb)
   }
 }
+
+/** @function */
+export const fromEquals = <A>(equals: (x: A, y: A) => boolean): Setoid<A> => ({
```

Comment:
The function `fromEquals` looks useless, could you please remove it

Reply:
Ok made the changes.